doc/authoring_command_modules/README.md

```diff
@@ -212,7 +212,6 @@ e.g.:
 Release History
 ===============
 
-0.0.3
```

Comment:
any story?

Reply:
Customers do not need to add version number for next release. We will add it during release.

Reply:
Sound reasonable. I prefer adding the next release info into history when we finish the current one.